Tasks/PublishTestResultsV1/package.json

```diff
@@ -18,6 +18,6 @@
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
     "globule": "^1.0.0",
-    "vsts-task-lib": "0.7.4"
+    "azure-pipelines-task-lib": "^2.9.3"
```

Comment:
Have we tested on any impact of upgrading this package ?

Reply:
As per this [issue](https://mseng.visualstudio.com/AzureDevOps/_workitems/edit/1670741), We should get warning **"Warning: Use Cipheriv for counter mode of aes-256-ct"** if we are using **vsts-task-lib : 0.7.4**, But we are getting **##[warning]Task 'PublishTestResults' (1.0.49) is using deprecated task execution handler. The task should use the supported task-lib: https://aka.ms/tasklib**. After upgrading i have tested and again getting the same warning.

Reply:
@v-padiseshu Is the publish functionality working as intended after the patch? did you check tests show up in test tab and the config for PTRv1 task all work?

Reply:
@suyash691 Publish functionality is working as intended.
Tests are showing up in the test tab as expected.
Need more information on the last question **config for PTRv1 task all work**

Reply:
Basically validate all the inputs in the UI configuration when you add PTRv1 task to your pipeline. And see that all inputs work as intended

Reply:
@suyash691  Yeah, I published this task and tested. They are working as intended